src/components/transactions/LiAnyTransaction.vue

```diff
@@ -10,7 +10,7 @@
         :time="transaction.time"
         :block="Number(transaction.height)"
         :memo="transaction.tx.value.memo"
-        :hide-data="hideData"
+        :hide-right-side="hideRightSide"
```

Comment:
What is the right side, and why are we hiding it?

Reply:
It's the right side of the transaction components that displays block #, Time/date etc. It adds unnecessary noise to the extension IMO

Reply:
Sure, Perhaps a better more descriptive name. `hide-right-side` doesn't tell me much if I'm not familiar with the component.

Perhaps `block-data`, `meta-data`…

Reply:
Sure, but what data is it blocking would come to mind with those names. Let's chat in the call tomorrow.

Reply:
sorry guys, i think this was my suggestion 

yeah, let's chat about it tomorrow.